source/agent/addons/quic/QuicTransportServer.h

```diff
@@ -8,14 +8,15 @@
 #define QUIC_QUICTRANSPORTSERVER_H_
 
 #include "QuicTransportConnection.h"
-#include "owt/quic/web_transport_server_interface.h"
+#include "owt/quic/quic_transport_server_interface.h"
```

Comment:
In the latest QUIC SDK, quic_transport_server_interface has been renamed to web_transport_server_interface.